infrastructure/comms/src/connection/i2p/address.rs

```diff
@@ -0,0 +1,36 @@
+//  Copyright 2018 The Tari Project
```

Comment:
```suggestion
//  Copyright 2019 The Tari Project
```

Reply:
Thanks - added `${today.year}` to my copyright profile on idea